automation/service/workflow.go

```diff
@@ -572,6 +577,10 @@ func (svc *workflow) Exec(ctx context.Context, workflowID uint64, p types.Workfl
 				return nil, err
 			}
 
+			if len(tt) == 0 {
+				return nil, nil
+			}
```

Comment:
perhaps move the above comment about skipping triggers to here and elaborate a bit more on why